pkg/api/frontendsettings.go

```diff
@@ -401,52 +416,66 @@ func getPanelSort(id string) int {
 	return sort
 }
 
-// EnabledPlugins represents a mapping from plugin types (panel, data source, etc.) to plugin IDs to plugins
+type availablePluginDTO struct {
```

Comment:
Instead of adding an `Enabled` property to PluginDTO i created an internal struct that contains both of them. The reason for this is that `PluginDTO` is used on multiple places in the project and the enabled value is located in `pluginsettings` so that would require us to read settings on all places where we use the PluginDTO to fill the enabled value.

Reply:
Make sense :+1: